CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2021-07-20 –– Organization Reorganization
```

Comment:
FYI, while the GitHub markdown renderer appears to turn `--` into a nice em dash, whatever processes it for the website doesn't:

![image](https://user-images.githubusercontent.com/16724/126562958-682a2742-ef04-45f5-8fd6-b11ac95d891d.png)